types/v1/tx.go

```diff
@@ -120,45 +125,34 @@ type (
 
 	// Transfer describes the transfer of currency
 	Transfer struct {
-		Value   Amount     `json:"value"`
-		Asset   string     `json:"asset"`
-		Memo    string     `json:"memo,omitempty"`
-		Inputs  []TxOutput `json:"inputs,omitempty"`
-		Outputs []TxOutput `json:"outputs,omitempty"`
+		Value Amount `json:"value"`
+		Asset string `json:"asset"`
 	}
 
 	// Delegation describes the blocking of a stacked asset
 	Delegation struct {
```

Comment:
one suggestion to use `Transfer` for all staking types, since they are both compatible with `Transfer`